Integrations/ESPHome/Core.yaml

```
@@ -218,6 +216,22 @@ text_sensor:
     id: wakeup_button_pressed
     name: "Wake-up Button Pressed"
     icon: "mdi:gesture-tap-button"
+  - platform: wifi_info
+    ip_address:
+      name: "IP Address"
+      id: wifi_ip
+  - platform: version
+    name: "ESPHome Version"
+    hide_timestamp: true
+    entity_category: "diagnostic"
+    entity_category: "diagnostic"
```

YAMLlint (1.38.0)

[error] duplication of key "entity_category" in mapping

(key-duplicates)